test/browser-tests/cache-first/cache-first.js

```diff
@@ -0,0 +1,103 @@
+/*
+  Copyright 2016 Google Inc. All Rights Reserved.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+*/
+
+/* eslint-env browser, mocha */
+
+'use strict';
+
+var swUtils = window.goog.SWHelper;
+
+describe('Test toolbox.cacheFirst', function() {
+  const serviceWorkersFolder = '/test/browser-tests/cache-first/serviceworkers';
+
+  it('should retrieve the first value from the cache and not update it', function() {
+    var iframe;
```

Comment:
Friendly reminder to switch out these `var` statements since we're using `const` and `let` elsewhere.